web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <!-- eslint-disable -->
+  <div>
+    <div v-for="item in children" class="item">
+      <div>
+        <i
+          v-if="item.isFile"
+          :class="['icon', 'file', 'status', getDiffType(item.file.Type), 'tooltip']"
+          :data-content="getDiffType(item.file.Type)"
+          data-position="right center"
+          ></i>
+        <a
+          v-if="item.isFile"
+          class="file mono tooltip"
+          :data-content="item.Name"
+          :href="item.isFile ? '#diff-' + item.file.NameHash : ''"
+          >{{ item.name }}</a
+        >
+        <div v-if="!item.isFile">
+          <i class="folder icon"></i>{{ item.name }}
```

Comment:
Please use SVG icons, e.g. `import {SvgIcon} from '../svg.js';`, see other Vue components for exact usage. Also, color should match the ones in repo file view, e.g. `color: var(--color-primary);` for directories.

Reply:
now they render as color-primary:

![image](https://user-images.githubusercontent.com/1135157/188213016-d7521c6a-1aa1-4e51-b352-f251d5fec2d5.png)

Reply:
We should really not render different icons and colors then what the file view does, that's just inconsistent. Please just copy what GitHub has with right-aligned diff icon and regular folder/file icons :)

Reply:
Style has been adjusted to use same colors and icons as file view and status is the github style

![image](https://user-images.githubusercontent.com/1135157/188225738-40915f40-4823-4395-8be1-efc3575e9348.png)